pandas/core/missing.py

```diff
@@ -99,6 +103,18 @@ def mask_missing(arr: ArrayLike, values_to_mask) -> np.ndarray:
     return mask
 
 
+def try_clean_fill_method(method):
```

Comment:
this is a bitty smelly, can you just use clean_fill_method?

Reply:
sure

Reply:
updated + greenish